src/components/Contributors.js

```diff
@@ -0,0 +1,36 @@
+import React, { useState, useEffect } from "react"
+import {
+  TiledLink,
+  TiledLinkContainer,
+  TiledLinkIcon,
+  SectionTitle,
+} from "../theme"
+
+const Contributors = () => {
+  const [contributors, setContributors] = useState(null)
+
+  useEffect(() => {
```

Comment:
We shouldn't be making dynamic calls. api fetch should be done at gatsby build time, so the data could be cached statically.

See details at #15